style.css

```diff
@@ -189,13 +203,44 @@ footer {
 	padding-bottom: 50px;
 }
 
+.under-construction {
+	width: 40%;
+}
+
+/*portfolio*/
+
+/*footer*/
+
+footer {
```

Comment:
I think your footer images scale a little too small. In fact I'm not sure you need them to resize at all as the screen shrinks: 
![image](https://user-images.githubusercontent.com/45392468/149965661-e3b5294e-beec-44a0-811c-3e6ee4320bc7.png)